crates/agentgateway/src/types/agent_xds.rs

```diff
@@ -625,6 +625,15 @@ impl TryFrom<&proto::agent::PolicySpec> for Policy {
 				Policy::ExtAuthz(http::ext_authz::ExtAuthz {
 					target: Arc::new(target),
 					context: Some(ea.context.clone()),
+					metadata_namespace: None,
```

Comment:
Can we add these to the xds protos as well? https://github.com/agentgateway/agentgateway/blob/main/crates/agentgateway/proto/resource.proto#L364